functions-methods.js

```diff
@@ -20,7 +24,17 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail(eMail) {
+    if (eMail.substring(eMail.indexOf("@")) === "@novi-education.nl")
```

Comment:
Ik zou de substring lostrekken naar een variabele, dan hoef je die niet 2x te doen
```suggestion
    const domain = eMail.substring(eMail.indexOf("@"));
    if (domain === "@novi-education.nl")
```